src/pages/profiles/ProfileDetail.tsx

```diff
@@ -38,16 +34,32 @@ const ProfileDetail: FC = () => {
   const {
     data: profile,
     error,
-    isLoading,
+    isLoading: isProfileLoading,
   } = useQuery({
     queryKey: [queryKeys.profiles, "detail", name],
     queryFn: () => fetchProfile(name, project),
   });
 
+  const {
+    data: projectObj,
+    error: projectError,
+    isLoading: isProjectLoading,
+  } = useQuery({
+    queryKey: [queryKeys.projects, project],
+    queryFn: () => fetchProject(project),
+  });
+
   if (error) {
     notify.failure("Loading profile failed", error);
   }
 
+  if (projectError) {
+    notify.failure("Loading project failed", error);
+  }
+  const isLoading = isProfileLoading || isProjectLoading;
+
+  const featuresProfiles = projectObj?.config["features.profiles"] === "true";
```

Comment:
You'd still need to pass around the `projectObj` though, right?

Reply:
True, we don't pass that in all cases atm, so it might not be much better.

Reply:
Yes, it would probably be even worse, because then you'd need to call the util function in every child component where you need that information. So maybe we can keep it as it is?